### PR TITLE
[token-cli] sequentially process confidential transfer withdraw in token-cli

### PR DESCRIPTION
**Pull request by samkim-crypto**
#### Problem
We can't do confidential token withdrawal through the cli due to the transaction size limit

#### Summary of changes
Divide up the process of creating proof context states and the actual token withdraw.

### CODE REVIEW

File: token/cli/src/main.rs [hunk 1]
```diff
@@ -3409,18 +3409,48 @@ async fn command_deposit_withdraw_confidential_tokens(
...
+            // close context state account
+            let context_state_authority_pubkey = context_state_authority.pubkey();
+            let close_context_state_signers = &[context_state_authority];
+            token
+                .confidential_transfer_close_context_state(
+                    &context_state_pubkey,
+                    &token_account_address,
+                    &context_state_authority_pubkey,
+                    close_context_state_signers,
+                )
```
Comment (joncinque) [inline token/cli/src/main.rs:3453]: Is it possible to do the withdraw and close in the same transaction? If so, it would be good to do it in this PR

**Review by joncinque — APPROVED** [review]
Great work! Just one suggestion to speed up the withdrawal to go down to 3 dependent transactions instead of 4

File: token/cli/src/main.rs (hunk 1 above)
Comment (samkim-crypto) [reply token/cli/src/main.rs:3453]: Yes, good idea!

Comment (samkim-crypto) [reply token/cli/src/main.rs:3453]: Added issue https://github.com/solana-labs/solana-program-library/issues/5685, which will be addressed in a follow-up PR.

**Pull request merged**